package.json

```diff
@@ -38,10 +38,7 @@
     "build": "stripes build --output ./output",
     "lint": "eslint .",
     "start": "stripes serve",
-    "test:unit": "jest --ci --coverage --passWithNoTests",
-    "test:e2e": "stripes test karma --karma.singleRun --karma.browsers ChromeDocker --karma.reporters mocha junit --coverage",
-    "test:e2e:dev": "stripes test karma --karma.browsers ChromeDocker",
-    "test": "yarn run test:e2e && yarn run test:unit"
+    "test": "jest --ci --coverage"
```

Comment:
I don't think there is any final decision how to setup jest/rtl. AFAIK we should wait until @funkymalc set up CI for coverage both tests

Reply:
I agree, but his changes should work when module doesn't have bigtest at all
after his updates npm command can be changed, no issues here